Jenkinsfile

```diff
@@ -346,6 +346,12 @@ pipeline {
                         buildHipClangJobAndReboot(build_type: 'debug', config_targets: Smoke_targets)
                     }
                 }
+                stage('Fp32 OpenCL Debug gfx1030') {
+                    agent{ label rocmnode("navi21") }
+                    steps{
+                        buildHipClangJobAndReboot(compiler: 'g++', build_type: 'debug', config_targets: Smoke_targets, build_env: extra_log_env, gpu_arch: "gfx1030")
```

Comment:
build**HipClang**JobAndReboot() is a little bit distracting here

Reply:
[Quality] Absolutely :face_with_head_bandage:. It worth refactoring this legacy stuff, but this is out of the scope of this PR.

/cc @shurale-nkn